src/main/java/com/launchdarkly/client/dynamodb/DynamoDbFeatureStoreBuilder.java

```diff
@@ -1,50 +1,44 @@
 package com.launchdarkly.client.dynamodb;
 
+import com.launchdarkly.client.Components;
 import com.launchdarkly.client.FeatureStore;
 import com.launchdarkly.client.FeatureStoreCacheConfig;
 import com.launchdarkly.client.FeatureStoreFactory;
-import com.launchdarkly.client.utils.CachingStoreWrapper;
+import com.launchdarkly.client.LDConfig;
+import com.launchdarkly.client.integrations.DynamoDbDataStoreBuilder;
+import com.launchdarkly.client.integrations.PersistentDataStoreBuilder;
+import com.launchdarkly.client.interfaces.DiagnosticDescription;
+import com.launchdarkly.client.value.LDValue;
 
 import java.net.URI;
 
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
 import software.amazon.awssdk.core.client.config.ClientOverrideConfiguration;
 import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
-import software.amazon.awssdk.services.dynamodb.DynamoDbClientBuilder;
 
 /**
- * Builder/factory class for the DynamoDB feature store.
+ * Deprecated builder class for the Redis-based persistent data store.
```

Comment:
I think this should say "DynamoDB-based" or the like rather than "Redis-based"

Reply:
Copypasta strikes again.